elixir/apps/domain/lib/domain/events/hooks/accounts.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Domain.Events.Hooks.Accounts do
+  def insert(_data) do
```

Comment:
```suggestion
  def on_insert(_data) do
```

YMMV but these are basically event-handlers, right? So perhaps prefix them with `on_`?

Reply:
Yeah perhaps will revisit in a later PR.